<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogEntryView</class>
 <widget class="QWidget" name="LogEntryView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Entry view</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="lblTime">
            <property name="text">
             <string>Time</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtTime"/>
          </item>
          <item>
           <widget class="QLabel" name="lblFunction">
            <property name="text">
             <string>Function</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtFunction"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="lblProcessId">
            <property name="text">
             <string>Process ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtProcessID"/>
          </item>
          <item>
           <widget class="QLabel" name="lblThreadID">
            <property name="text">
             <string>Thread ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtThreadID"/>
          </item>
          <item>
           <widget class="QLabel" name="lblTraceLevel">
            <property name="text">
             <string>Trace Level</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtTraceLevel"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="lblDescription">
            <property name="text">
             <string>Description</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEdit"/>
          </item>
          <item alignment="Qt::AlignRight">
           <widget class="QPushButton" name="btnClose">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Close</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
